lib/stat_tracker.rb

```diff
@@ -212,6 +212,18 @@ def team_id_to_team_name(id)
     end
   end
 
+  def total_wins(game_teams_filtered = @game_teams)
+    game_teams_filtered.count do |gameteam|
+      gameteam.result == "WIN"
+    end
+  end
+
+  def filter_by_teamid(id)
+    @game_teams.select do |game_team|
+      game_team.team_id == id
+    end
+  end
```

Comment:
Be specific with id argument; change to team_id (as opposed to home_id or away_id)